Lib/test/test_json/test_tool.py

```diff
@@ -11,7 +11,7 @@ class TestTool(unittest.TestCase):
     data = """
 
         [["blorpie"],[ "whoops" ] , [
-                                 ],\t"d-shtaeou",\r"d-nthiouh",
+                                 ],\t"d-shtaeou",\r"🐍 and δ",
```

Comment:
Okay in https://github.com/python/cpython/pull/201/commits/d69907079b5a9e9e6b6de49245cb0e686ead39ff I use the following unicode characters, which should cover three ranges:

+ δ U+03B4 `\u03B4`
+ 🐍 U+1F40D `\N{snake}`
+ 𝀷 U+1D037 `\U0001D037`